README.md

```diff
@@ -32,13 +33,34 @@ ota_password: "<some random password>"
 api_password: "<either: your home assistant api password>"
 ```
 
+Then modify the `example.yaml` file to suite your needs or use your **own** configuration and just import the `nebula_light_device.yaml` esphome package with a `device_name` variable defined:
+```yaml
+packages:
+  nebula: !include
+    file: nebula_light_device.yaml
+    vars:
+      device_name: "Star Projector"
+```
+
+You can also import the esphome nebula package from a remote git repository by using:
+```yaml
+substitutions:
+  device_name: "Star Projector"
+
+packages:
+  remote_package:
+    url: https://github.com/seberm/esphome-SK20-Nebula-Light.git
```

Comment:
I will keep this `url` and `ref` for now just for the testing. I will change it to the original repo before the merge.

Reply:
This is now resolved - I've changed the configuration to point to a main repository.